Traceback (most recent call last):
  File "C:\Users\vania wrenda\AppData\Local\Programs\Python\Python312\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\vania wrenda\AppData\Local\Programs\Python\Python312\Lib\site-packages\nbclient\client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vania wrenda\AppData\Local\Programs\Python\Python312\Lib\site-packages\jupyter_core\utils\__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vania wrenda\AppData\Local\Programs\Python\Python312\Lib\asyncio\base_events.py", line 685, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\vania wrenda\AppData\Local\Programs\Python\Python312\Lib\site-packages\nbclient\client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "C:\Users\vania wrenda\AppData\Local\Programs\Python\Python312\Lib\site-packages\nbclient\client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\vania wrenda\AppData\Local\Programs\Python\Python312\Lib\site-packages\nbclient\client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
#Mengonversi kolom Harga Beras dalam dataframe mnenjadi data float
df['Harga Beras'] = pd.to_numeric(df['Harga Beras'].str.replace(',', ''), errors='coerce')
print(df)
------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mAttributeError[0m                            Traceback (most recent call last)
Cell [1;32mIn[8], line 2[0m
[0;32m      1[0m [38;5;66;03m#Mengonversi kolom Harga Beras dalam dataframe mnenjadi data float[39;00m
[1;32m----> 2[0m df[[38;5;124m'[39m[38;5;124mHarga Beras[39m[38;5;124m'[39m] [38;5;241m=[39m pd[38;5;241m.[39mto_numeric([43mdf[49m[43m[[49m[38;5;124;43m'[39;49m[38;5;124;43mHarga Beras[39;49m[38;5;124;43m'[39;49m[43m][49m[38;5;241;43m.[39;49m[43mstr[49m[38;5;241m.[39mreplace([38;5;124m'[39m[38;5;124m,[39m[38;5;124m'[39m, [38;5;124m'[39m[38;5;124m'[39m), errors[38;5;241m=[39m[38;5;124m'[39m[38;5;124mcoerce[39m[38;5;124m'[39m)
[0;32m      3[0m [38;5;28mprint[39m(df)

File [1;32m~\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\core\generic.py:6299[0m, in [0;36mNDFrame.__getattr__[1;34m(self, name)[0m
[0;32m   6292[0m [38;5;28;01mif[39;00m (
[0;32m   6293[0m     name [38;5;129;01mnot[39;00m [38;5;129;01min[39;00m [38;5;28mself[39m[38;5;241m.[39m_internal_names_set
[0;32m   6294[0m     [38;5;129;01mand[39;00m name [38;5;129;01mnot[39;00m [38;5;129;01min[39;00m [38;5;28mself[39m[38;5;241m.[39m_metadata
[0;32m   6295[0m     [38;5;129;01mand[39;00m name [38;5;129;01mnot[39;00m [38;5;129;01min[39;00m [38;5;28mself[39m[38;5;241m.[39m_accessors
[0;32m   6296[0m     [38;5;129;01mand[39;00m [38;5;28mself[39m[38;5;241m.[39m_info_axis[38;5;241m.[39m_can_hold_identifiers_and_holds_name(name)
[0;32m   6297[0m ):
[0;32m   6298[0m     [38;5;28;01mreturn[39;00m [38;5;28mself[39m[name]
[1;32m-> 6299[0m [38;5;28;01mreturn[39;00m [38;5;28;43mobject[39;49m[38;5;241;43m.[39;49m[38;5;21;43m__getattribute__[39;49m[43m([49m[38;5;28;43mself[39;49m[43m,[49m[43m [49m[43mname[49m[43m)[49m

File [1;32m~\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\core\accessor.py:224[0m, in [0;36mCachedAccessor.__get__[1;34m(self, obj, cls)[0m
[0;32m    221[0m [38;5;28;01mif[39;00m obj [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[0;32m    222[0m     [38;5;66;03m# we're accessing the attribute of the class, i.e., Dataset.geo[39;00m
[0;32m    223[0m     [38;5;28;01mreturn[39;00m [38;5;28mself[39m[38;5;241m.[39m_accessor
[1;32m--> 224[0m accessor_obj [38;5;241m=[39m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_accessor[49m[43m([49m[43mobj[49m[43m)[49m
[0;32m    225[0m [38;5;66;03m# Replace the property with the accessor object. Inspired by:[39;00m
[0;32m    226[0m [38;5;66;03m# https://www.pydanny.com/cached-property.html[39;00m
[0;32m    227[0m [38;5;66;03m# We need to use object.__setattr__ because we overwrite __setattr__ on[39;00m
[0;32m    228[0m [38;5;66;03m# NDFrame[39;00m
[0;32m    229[0m [38;5;28mobject[39m[38;5;241m.[39m[38;5;21m__setattr__[39m(obj, [38;5;28mself[39m[38;5;241m.[39m_name, accessor_obj)

File [1;32m~\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\core\strings\accessor.py:191[0m, in [0;36mStringMethods.__init__[1;34m(self, data)[0m
[0;32m    188[0m [38;5;28;01mdef[39;00m [38;5;21m__init__[39m([38;5;28mself[39m, data) [38;5;241m-[39m[38;5;241m>[39m [38;5;28;01mNone[39;00m:
[0;32m    189[0m     [38;5;28;01mfrom[39;00m [38;5;21;01mpandas[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcore[39;00m[38;5;21;01m.[39;00m[38;5;21;01marrays[39;00m[38;5;21;01m.[39;00m[38;5;21;01mstring_[39;00m [38;5;28;01mimport[39;00m StringDtype
[1;32m--> 191[0m     [38;5;28mself[39m[38;5;241m.[39m_inferred_dtype [38;5;241m=[39m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_validate[49m[43m([49m[43mdata[49m[43m)[49m
[0;32m    192[0m     [38;5;28mself[39m[38;5;241m.[39m_is_categorical [38;5;241m=[39m [38;5;28misinstance[39m(data[38;5;241m.[39mdtype, CategoricalDtype)
[0;32m    193[0m     [38;5;28mself[39m[38;5;241m.[39m_is_string [38;5;241m=[39m [38;5;28misinstance[39m(data[38;5;241m.[39mdtype, StringDtype)

File [1;32m~\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\core\strings\accessor.py:245[0m, in [0;36mStringMethods._validate[1;34m(data)[0m
[0;32m    242[0m inferred_dtype [38;5;241m=[39m lib[38;5;241m.[39minfer_dtype(values, skipna[38;5;241m=[39m[38;5;28;01mTrue[39;00m)
[0;32m    244[0m [38;5;28;01mif[39;00m inferred_dtype [38;5;129;01mnot[39;00m [38;5;129;01min[39;00m allowed_types:
[1;32m--> 245[0m     [38;5;28;01mraise[39;00m [38;5;167;01mAttributeError[39;00m([38;5;124m"[39m[38;5;124mCan only use .str accessor with string values![39m[38;5;124m"[39m)
[0;32m    246[0m [38;5;28;01mreturn[39;00m inferred_dtype

[1;31mAttributeError[0m: Can only use .str accessor with string values!

